roofit/hs3/test/histfactory.py

```diff
@@ -0,0 +1,178 @@
+import unittest
+import ROOT
+
+
+class HistFactory(unittest.TestCase):
```

Comment:
```suggestion
class TestHS3HistFactoryJSON(unittest.TestCase):
```
`HistFactory` is not a good name for a test class.